### ISSUE TITLE
Firewalld 0.5.0(-1) not working properly on Arch Linux and other  Distros

### ISSUE THREAD

**Issue by Rudi3** [firewalld/firewalld#287]
As seen in [Unix.Stackexchange](https://unix.stackexchange.com/questions/419918/firewalld-failed-to-apply-rules-set-rule-failure-when-reload).

Version: firewalld-0.5.0-1.
**I get this when starting the daemon:**
```
ERROR: Failed to apply rules. A firewall reload might solve the issue if the firewall has been modified using ip*tables or ebtables.
ERROR: set_rule() missing 1 required positional argument: 'log_denied'
ERROR: INVALID_ZONE
Stopping firewalld - dynamic firewall daemon... (I might have restarted it)
ERROR: Failed to apply rules. A firewall reload might solve the issue if the firewall has been modified using ip*tables or ebtables.
ERROR: set_rule() missing 1 required positional argument: 'log_denied'
ERROR: Failed to apply rules. A firewall reload might solve the issue if the firewall has been modified using ip*tables or ebtables.
ERROR: set_rule() missing 1 required positional argument: 'log_denied'

```
**And this happens when reloading as suggested:**
`Error: 'FirewallZoneIPTables' object has no attribute '_FirewallZone__icmp_block_inversion'
`

I solved it by downgrading to a lower version. Then it worked as expected.

It may have happened in one of these commits, I didn't find anything so far...
[56a26d4eca4d2085644b8752be294e4a43ecf794](https://github.com/firewalld/firewalld/commit/56a26d4eca4d2085644b8752be294e4a43ecf794)
[3d042932faa0a1e81859465d515edccf69d637f8](https://github.com/firewalld/firewalld/commit/3d042932faa0a1e81859465d515edccf69d637f8)

Also a link to the diff if anyone wants to look into it:
[v0.4.4.6...v0.5.0](https://github.com/firewalld/firewalld/compare/v0.4.4.6...v0.5.0)

**Comment by alucryd** [comment]
Associated Arch Linux bug report: https://bugs.archlinux.org/task/57255

**Comment by alucryd** [comment]
Thanks for the quick fix!

**Comment by Rudi3** [comment]
Yes thanks a lot!

**Comment by erig0** [comment]
@alucryd, @Rudi3,
I'll try to tag a new release soon. I have one more thing I want to look at first.

**Comment by alucryd** [comment]
Got it, thanks. For the time being I backported this patch into the Arch Linux package.

**Comment by Anidetrix** [comment]
@Rudi3 Same happens for me, issue is not resolved

**Comment by erig0** [comment]
@Rudi3, @Anidetrix,
I fixed that with the same push. Sorry I forgot to mention it.  6796e43.

**Comment by erig0** [comment]
Tagged and released v0.5.1. https://github.com/firewalld/firewalld/releases/tag/v0.5.1
